client/src/variable.scss

```
@@ -47,4 +47,13 @@ $fz-medium-2: 2.4rem;
 
 $fz-large-1: 3.6rem;
 $fz-large-2: 4.1rem;
-$fz-large-3: 4.4rem;
\ No newline at end of file
+$fz-large-3: 4.4rem;
+
+// Variablen für Tags
+// a-tags color
+$ac: #777777;
+
+// Variablen für Font-Weights
+$fw-medium: 400;
+$fw-regular: 500;
+$fw-bold: 700;
```

Hier müsstest du noch die `$fw-light`hinzufügen.
Und Medium und regular sind vertauscht:
```suggestion
// Variablen für Font-Weights
$fw-light: 300;
$fw-regular: 400;
$fw-medium: 500;
$fw-bold: 700;
```

---

done! das mit regular und medium checke ich nicht, dritte mal schon korrigiert. der kleine möchte doch groß bleiben wahrscheinlich. ist erledigt ;) ...hoffentlich diesmal